### DISCUSSION TITLE
v0.1.2 - Easy Launcher

### DISCUSSION THREAD

**Discussion by CreativeCodeCat** [DroidWorksStudio/EasyLauncher#58] [Check for Updates]
# What's Changed

### Implemented Enhancements:

- Updated the app icon. - ([1e1b81c](https://github.com/DroidWorksStudio/EasyLauncher/commit/1e1b81cf83341e28c43efb788a86e5d99d061ef4))
- Added custom gestures. - ([924e052](https://github.com/DroidWorksStudio/EasyLauncher/commit/924e052598c9f4c8cd990a2d9e0cdb73c4e1dcfa))

### Bug Fixes:

- Fixed proguard building - ([80ecd90](https://github.com/DroidWorksStudio/EasyLauncher/commit/80ecd906e731b14b276944fb3ee07578fd079822))
- Fixed a workflow so it works correctly - ([773a2f5](https://github.com/DroidWorksStudio/EasyLauncher/commit/773a2f546f218f040ce6cf2666692d18818b65d1))

### Dependency Updates:

- Updated Dependencies - ([8e796ca](https://github.com/DroidWorksStudio/EasyLauncher/commit/8e796ca2fe95aa039873c5b47db7ed50dafdbacc))

### Housekeeping:

- Moved around a few files to be cleaner. - ([0677e37](https://github.com/DroidWorksStudio/EasyLauncher/commit/0677e3792cfa662625010d4fa733ae5e87318e54))

### Language Support:

- Updated Language Files. ([#52](https://github.com/DroidWorksStudio/EasyLauncher/issues/52)) - ([e73fabb](https://github.com/DroidWorksStudio/EasyLauncher/commit/e73fabb85a739a358867741fc696b10a05ed8670))
- Updated Language Files. ([#51](https://github.com/DroidWorksStudio/EasyLauncher/issues/51)) - ([8c4ac73](https://github.com/DroidWorksStudio/EasyLauncher/commit/8c4ac73bbf541c4d21c8d403556761326752cd40))
- Updated Language Files. ([#50](https://github.com/DroidWorksStudio/EasyLauncher/issues/50)) - ([81d2242](https://github.com/DroidWorksStudio/EasyLauncher/commit/81d2242b9715d0b47c72391cf985bf7e2359161b))

### Refactoring:

- More widget stuff and fixing of files. - ([327f144](https://github.com/DroidWorksStudio/EasyLauncher/commit/327f1447a77a4fe9f18aeaa6a98700e85e3d25bc))
- Moved the getActionType function into AppHelper class. - ([48b2913](https://github.com/DroidWorksStudio/EasyLauncher/commit/48b2913797a46857df9ec89348bb1b9e01164c81))
- Added widget page to the gestures. - ([94fe747](https://github.com/DroidWorksStudio/EasyLauncher/commit/94fe747ffdfc869237c1dd200966cda8e2b2fe23))

### Reverts:

- Revert "chore(deps): bump androidx.biometric:biometric-ktx ([#53](https://github.com/DroidWorksStudio/EasyLauncher/issues/53))" ([#54](https://github.com/DroidWorksStudio/EasyLauncher/issues/54)) - ([623a353](https://github.com/DroidWorksStudio/EasyLauncher/commit/623a353de56aabf7cf7c4c2cf89ecd80f79a00ff))

**Full Changelog**: https://github.com/DroidWorksStudio/EasyLauncher/compare/0.1.1...0.1.2

<hr /><em>This discussion was created from the release <a href='https://github.com/DroidWorksStudio/EasyLauncher/releases/tag/0.1.2'>v0.1.2 - Easy Launcher</a>.</em>

**Comment by cankurttekin** [comment]
thanks for your effort. but the animations are so so long. it is unusable for me. it gives dizziness after couple minutes of playing with gestures.

**Reply by CreativeCodeCat** [reply]
hey i will reduce the amount of time animations take.
